05/app.js

```diff
@@ -1,53 +1,67 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    p1: 0,
+  },
+  links: {
+    "/dolor.html": 0,
+  },
 };
 
 /* tutaj umieść swój kod */
+const everyP = Array.from(document.querySelectorAll("p"));
 
+everyP.forEach(function (element) {
+  element.addEventListener("click", function (e) {
+    if (e.target.getAttribute("data-id") === "p1") {
+      stats.paragraphs["p1"]++;
+    } else if (e.target.getAttribute("href") === "/dolor.html") {
```

Comment:
To nie miało być "sztywne" rozwiązanie. Kliknięcie każdego `p` czy `e` ma uzupełnić statystykę. Czyli jeśli klikamy po raz pierwszy w `p2` to ma się pojawić `p2 -> 1`. Postaraj się poprawić :)